src/System.Configuration.ConfigurationManager/tests/System/Configuration/SubclassTypeValidatorAttributeTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Configuration;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class SubclassTypeValidatorAttributeTests
+    {
+        [Fact]
+        public void GetBaseClass_PassValidTypeToConstructor()
+        {
+            SubclassTypeValidatorAttribute attribute = new SubclassTypeValidatorAttribute(typeof(string));
+            Type test = attribute.BaseClass;
+            Assert.Equal(test, typeof(string));
```

Comment:
http://whatis.techtarget.com/definition/nit
Basically something small being pointed out, not a big deal, but something to be improved